### DISCUSSION TITLE
Adding OOP Background Task to WinUI3 / WindowsAppSDK App

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/WindowsAppSDK#2859] [Q&A] [answered]
I'm converting a UWP app to WinUI3/WindowsAppSDK and converting the OOP background task with the help of this sample: https://github.com/microsoft/CsWinRT/tree/master/src/Samples/BgTaskComponent

I have several errors similar to the one below. How do I fix this?

````
Severity	Code	Description	Project	File	Line	Suppression State
Error	APPX1101	Payload contains two or more files with the same destination path 'WinRT.Host.runtimeconfig.json'. Source files: 
C:\AppBackgroundTasks\WinRT.Host.runtimeconfig.json
C:\AppBackgroundTasks\bin\x64\Debug\net6.0-windows10.0.22621.0\WinRT.Host.runtimeconfig.json	AppWTS	C:\Users\hhhhh\.nuget\packages\microsoft.windowsappsdk\1.1.4\buildTransitive\Microsoft.Build.Msix.Packaging.targets	1479	

````

**Comment by GreenShadeZhang** [comment]
At present, I also report some errors when I use background tasks to package msix in wasdk, and I recommend that the official give a sample demo to ensure that the service can be used normally to ensure that it is not a WASDK bug.

**Comment by myokeeh** [comment]
Is there better guidance on this topic?

**Reply by myokeeh** [reply, marked as answer]
https://github.com/microsoft/CsWinRT/issues/1327
